runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -1281,8 +1286,12 @@ void addPCollectionRequiringIndexedFormat(PCollection<?> pcol) {
   }
 
   void maybeRecordPCollectionWithAutoSharding(PCollection<?> pcol) {
-    if (hasExperiment(options, "enable_streaming_auto_sharding")
-        && !hasExperiment(options, "beam_fn_api")) {
+    if (hasExperiment(options, "beam_fn_api")) {
```

Comment:
Non-portable is not ready either. Should we say that it's not supported in general and enable it when the backend is ready?

Reply:
Actually would it make more sense to put this warning in the transform default implementation?

Reply:
I think it makes the most sense here.